data_structures/stack/Infix_to_Postfix.C

```diff
@@ -0,0 +1,133 @@
+// C program to convert infix expression to postfix 
+#include <stdio.h> 
+#include <string.h> 
+#include <stdlib.h> 
+
+// Stack type 
+struct Stack 
+{ 
+	int top; 
+	unsigned capacity; 
+	int* array; 
```

Comment:
I think ``array`` should be of type ``char`` because it is used to store operators (which are characters) to avoid unnecessary implicit type conversion in pop and push.